xelahjs/xelah
# Add the beginnings of a menu

For https://github.com/xelahjs/xelah/issues/23
This is ready to review now. 

It adds a menu to the first example in styleguidist. It just uses simple buttons since each application will need to decide how to present the menu. The real useful bits of code will be in the src/actions/ folder. 

To test:
In the HtmlPerfEditor click the menu button then click on an editable paragraph. You should see the menu appear with \p and \q1 etc. Each should format the text as expected. 
Do the same in the introduction graph editor.
You should see many more options for \imt \is \ip \im \ipi \iot \io \io2 each should format the block appropriately.
